packages/api/src/controllers/region.js

```diff
@@ -14,7 +29,7 @@ app.get("/", async (req, res, next) => {
   regions.forEach((region) => {
     region.orchestrators.forEach((orch) => {
       orch.region = region.region;
-      flatOrchList.push(orch);
+      flatOrchList.push({ score: defaultScore, ...orch });
```

Comment:
Do we need to mutate the `orch` in-place? How about adding the `region` on this obj literal too?
`flatOrchList.push({ score: defaultScore, ...orch, region: region.region });`

Reply:
fixed